src/reporter/main.py

```diff
@@ -10,12 +10,12 @@
 import sys
 from pathlib import Path
 
-from . import reports
```

Comment:
i think something is up if you had to make this change. relative imports should be the correct way to import modules.